yo_trombinoscope_member/static/src/snippets/s_trombinoscope_member/000.js

```diff
@@ -1,12 +1,13 @@
 /** @odoo-module **/
 
 import publicWidget from "@web/legacy/js/public/public_widget";
+import { rpc } from "@web/core/network/rpc";
 
 const TrombinoscopeMember = publicWidget.Widget.extend({
     selector: '.trombinoscope-member',
     init: function () {
         this._super.apply(this, arguments);
-        this.rpc = this.bindService("rpc");
+        this.rpc = rpc;
```

Comment:
Why still defined this.rpc if use rpc directly ?

Reply:
thought it's the standard to attach it to `this`
i'll update it